spec/controllers/controller_patches_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "caching blog content" do
 
     before do
-      allow(controller).to receive(:blog).and_return("")
+      allow(controller).to receive(:get_blog_content)
```

Comment:
Is there a reason for removing `.and_return("")`? It seems like there'd be no point in this stub if we're not setting a return value.